src/constants/index.jsx

```diff
@@ -183,3 +187,318 @@ export const TILE_MAP_STATES = [
   'WV',
   'WY'
 ]
+
+export const STATE_DATA = [
```

Comment:
This is a nitpick, but we may want to put the source of the data in a comment above this block so future maintainers (us) will know how to update.

Reply:
Added, from admiral's comment here:
https://github.cfpb.gov/data-products-team/explorer-docs/issues/849#issuecomment-236310